src/inmanta/data/__init__.py

```diff
@@ -4408,6 +4409,9 @@ def table_name(cls) -> str:
 
     # Field based on content from the resource actions
     last_deploy: Optional[datetime.datetime] = None
+    # When a resource is updated in a new model version, it might take some time until this update reaches the scheduler.
+    # This is the attribute hash that the scheduler considers the last released attribute hash for the given resource.
+    current_intent_attribute_hash: Optional[str] = None
```

Comment:
HAS_UDPATE state, right. 


I would document that:
```
To avoid write contention, the `ComplianceStatus` is split up in different fields that are written from different code paths. see get_compliance_status for the associated logic 
```